src/util/CdnUris.js

```diff
@@ -14,11 +14,10 @@ export function getSwapPluginIconUri(pluginId: string, theme: Theme) {
 }
 
 /**
- * Swap Plugin Icons
+ * Liquidity/Reward Pool Icons
```

Comment:
We should just fully commit to "Stake Provider Icons" and not describe what this entity could be is here because if we extend what the entity means, then we'd have to somehow remember to update this comment.

Reply:
Good catch, missed this